src/pages/Wallet/WalletOverview/components/AvailableAssetsTable/AvailableAssetsTable.tsx

```diff
@@ -59,6 +65,24 @@ const AvailableAssetsTable = ({ balances, pooledTickers }: AvailableAssetsTableP
             textSize={isMobile ? 'base' : undefined}
             marginBottom={isMobile ? 'spacing4' : undefined}
             {...getCoinIconProps(currency)}
+            tooltip={
```

Comment:
this could be handled inside the `AssetCell` like the rest of the logic. This changes makes the code cleaner. You would have to bring this isFeeToken into the global utils folder.

Reply:
Not sure I follow—AssetCell is used elsewhere (e.g. PoolsTable), and we only want to show the tooltip on the wallet page.